md_api/src/main/java/com/wipro/ats/bdre/md/api/Lineage.java

```diff
@@ -31,11 +32,10 @@
 /**
  * Created by KA294215 on 17-12-2015.
  */
-public class Lineage {
-    private static final Logger LOGGER = Logger.getLogger(BatchEnqueuer.class);
+public class Lineage extends MetadataAPIBase {
+    private static final Logger LOGGER = Logger.getLogger(Lineage.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)